core/primer.md

```diff
@@ -2,6 +2,185 @@
+  extensions?
+
+Schema registries can provide an answer to these questions, but just like event
+brokers, there are multiple registries, such as the Confluent Schema Registry,
+the Azure Event Hubs Registry or the Apicurio Registry. When an event travels
+through multiple brokers, its schema has to be introduced to the broker's
+registry and clients later have to deal with the implementation differences
+between different registries, for example when trying to validate an event
+structure. This hinders the interoperability of event brokers CloudEvents had in
+mind. xRegistry therefore comes in with a similar motivation to CloudEvents:
+Harmonize another piece of technology in the messaging / eventing ecosystem to
+increase interoperability and decouple event handling from broker products and
+protocols.
+
+Even though xRegistry was built with eventing in mind, the concept of this
+registry specification is far more powerful. Take a look at the [Possible Use
+Cases](#possible-use-cases) for examples outside the eventing world.
+
+## Design Goals
+
+- **Interoperability:** Enable effortless discovery, validation and versioning
+  of resources.
+- **Standardization:** Establish a high level structure for managing resource
+  metadata behind which vendors can unite.
+- **Extensibility:** Accommodate future innovations by allowing for the
+  definition of custom attributes and extensions tailored to specific needs
+  – not only for the happy path, but errors as well.
+- **Flexibility:** Support various resource types, protocols (MQTT, AMQP, Kafka,
+  NATS, HTTP..) and schema languages (JSON Schema, Avro Schema, Protobuf..)
+  – not just CloudEvents.
+- **Simplicity:** Allow simple use cases that do not need versioning of
+  resources or a full blown schema registry on the server.
+
+### Non-Goals
+
+- **Authentication and Authorization:** Rely on established security standards
+  depending on the registry representation.
+- **Relationships between event channels:** Focus on precisely describing a
+  single event channel before standardizing the relationships between event
+  channels.
+
+## Representations
+
+An xRegistry can be represented in the different ways: a JSON file, a static
+file server and an API server. When building up this spec the API server had the
+highest priority, followed by the file representation and the static file server
+as an option to transition between the first two. To enable this transition, one
+of the design goals is the symmetry between all representations.
+
+Whenever registries get large, you will realistically want to use the API server
+or CLI to create the registry and then use its export function to generate files
+for the file or static file server representation.
+
+### File
+
+The registry is represented as a single JSON file.
+
+The primary use case for this representation is a registry which purpose is to
+describe a single project. It basically becomes a declaration manifest of the
```

Comment:
I'm not sure what the right word is but I don't think it's "project" since an xReg could describe many different projects (e.g. services). Perhaps it would be better to focus on why people would use "file" rather than what's in the registry. E.g. they want to work off-line, or check it into git (like you mention next), or for optimization when running the info thru tooling (e.g. one GET instead of lots) ....   since I think people would choose each of the 3 choices based on reasons other than "what's in the registry", and more because of how they want to access (or host) the data.

Reply:
I changed the paragraph, is this what you were looking for?

Reply:
yes - thanks!